package.json

```diff
@@ -46,7 +46,7 @@
     "eslint-plugin-notice": "^0.9.10",
     "eslint-plugin-prettier": "^3.3.1",
     "grpc_tools_node_protoc_ts": "5.2.2",
-    "grpc-tools": "1.11.1",
+    "grpc-tools": "1.11.3",
```

Comment:
I suggest to move this this version if it's possible to build the project on Apple silicon chip

Reply:
Makes sense to upgrade.